docs/migration/v8-to-v9.md

```diff
@@ -210,30 +233,34 @@ It will be removed in a future major version.
   });
   ```
 
-- The `DEFAULT_USER_INCLUDES` constant has been removed.
+- The `transactionContext` property on the `samplingContext` argument passed to the `tracesSampler` and `profilesSampler` options has been removed. All object attributes are available in the top-level of `samplingContext`.
```

Comment:
For me this change falls into the category: "we should highlight this somehow up top", as it cannot be caught nicely by linters (I think) and is very possible to actually affect users at runtime. 🤔

Reply:
True, and we also should add a code example I realized!